packages/docusaurus/src/webpack/plugins/ChunkAssetPlugin.ts

```diff
@@ -15,8 +15,10 @@ class ChunkAssetPlugin {
       /* We modify webpack runtime to add an extra function called "__webpack_require__.gca"
       that will allow us to get the corresponding chunk asset for a webpack chunk.
       Pass it the chunkName or chunkId you want to load.
-      For example: if you have a chunk named "my-chunk-name" that will map to "/0a84b5e7.c8e35c7a.js" as its corresponding output path
-      __webpack_require__.gca("my-chunk-name") will return "/0a84b5e7.c8e35c7a.js" */
+      For example: if you have a chunk named "my-chunk-name" that will map to "/publicPath/0a84b5e7.c8e35c7a.js" as its corresponding output path
+      __webpack_require__.gca("my-chunk-name") will return "/publicPath/0a84b5e7.c8e35c7a.js"
+      "gca" stands for "get chunk asset"
+      */
       mainTemplate.hooks.requireExtensions.tap(pluginName, (source, chunk) => {
```

Comment:
@alexander-akait was wondering if you could help me figure out what this `requireExtensions` hook does and if there is any doc?
We want to add a custom method `__webpack_rerquire__.gca` and it has been implemented here: https://github.com/facebook/docusaurus/pull/2118

But this hook seems to run 2  or 3 times during the compilation, so was wondering if we used it correctly?